packages/contract-tools/collect/schema-collector.test.ts

```diff
@@ -385,3 +386,82 @@ describe("collect", () => {
     });
   });
 });
+
+describe("PythonSchemaCollector", () => {
+  let tmpDir: string;
+  let manifestPath: string;
+  let collector: PythonSchemaCollector;
+  let previousCwd: string;
+
+  beforeEach(async () => {
+    previousCwd = process.cwd();
+
+    tmpDir = await mkdtemp(join(tmpdir(), "collect-python-test-"));
+    process.chdir(tmpDir);
+
+    manifestPath = join(tmpDir, "rejot-manifest.json");
+    await writeFile(manifestPath, JSON.stringify({ slug: "@test/", manifestVersion: 0 }));
```

Comment:
```suggestion
    await writeFile(manifestPath, JSON.stringify({ slug: "test-", manifestVersion: 0 }));
```

FYI I made a change that makes sure only alphanumeric and hyphens are allowed.